.vscode/settings.json

```diff
@@ -2,5 +2,6 @@
   "editor.tabSize": 2,
   "files.eol": "\n",
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "explorer.compactFolders": false
```

Comment:
not really necessary to include within this PR.

Reply:
this file should be excluded from the git history. It only contains personal changes. I will remove my change.

Reply:
The only reason I see that it should stay in there is, that it locks your typescript version of vscode to the one installed in node_modules, and not the one shipped with the editor. 
This can be a huge hassle when developing (because ts is behaving very differently in in minor version changes). 
I would keep the file in there, but please remove your change.